my-fund-backend/src/main/java/com/myfund/controller/AuthControler.java

```diff
@@ -0,0 +1,49 @@
+package com.myfund.controller;
+
+import com.myfund.model.DTO.CreateUserDTO;
+import com.myfund.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.FieldError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.HashMap;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/")
+public class AuthControler {
+
+    private final UserService userService;
+
+    @Autowired
+    public AuthControler(UserService userService) {
+        this.userService = userService;
+    }
+
+    @PostMapping("/register")
+    ResponseEntity<String> registereUser(@Valid @RequestBody CreateUserDTO createUserDTO) {
+        return ResponseEntity.ok("User is valid");
```

Comment:
is that ok? Register should create new user,

Reply:
added userService.createUser(createUserDTO);